x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/slm/SnapshotLifecyclePolicy.java

```diff
@@ -287,6 +339,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.startObject();
         builder.field(NAME.getPreferredName(), this.name);
         builder.field(SCHEDULE.getPreferredName(), this.schedule);
+        if (this.interval != null) {
+            builder.field(INTERVAL.getPreferredName(), this.interval);
+        }
```

Comment:
As it's currently written, if schedule is defined a GET request would not contain an interval field:
```
{
 "schedule": "* * * * * *",
...
}
```
whereas, if interval is defined, there would be a null (or maybe "null"?) schedule field:
```
{
 "schedule": null,
 "interval": "30m"
...
}
```
I dislike this asymmetry, but I prefer to not write out empty values. Unfortunately, we can't (easily) remove the schedule field, since this would be a breaking change.  The alternative would be to include a field with a null value in both cases. What do you think?

Reply:
We should cover that scenario with other breaking changes policy experts -- it's only a breaking change **if you use the new feature** and I'm not sure we consider those breaking changes automatically.

Reply:
Ah yeah I see what you mean. I think that makes sense that it would not be a breaking change in that case. I'll follow-up withe some bcc experts

Reply:
I'm resolving this part of the conversation because we're no longer tracking an `interval` under its own key.